packages/is-loki-running/package.json

```diff
@@ -16,7 +16,8 @@
   },
   "license": "MIT",
   "files": [
-    "src"
+    "src",
+    "*.d.ts"
```

Comment:
Should we also add `!*.d.ts` to the `.npmignore`? @oblador 

https://github.com/oblador/loki/blob/2d833f03b6f2695e0a5b6e5f30e7b21e5e924373/.npmignore#L1-L6

Reply:
Kinda weird to have both, I think that since the migration to a monorepo this file would be ignored